products/factfinder/pipelines/acs_manual_update.py

```diff
@@ -111,11 +113,33 @@ def transform_all_dataframes(year):
     return combined_df
 
 
-def filter_by_metadata(df, year):
```

Comment:
@damonmcc do you know why this was being done before? Do they only want a subset of fields in AE? I'll follow up with Tim/Tyler too

Reply:
The new acs data is <10% larger than the last one, so this isn't hugely consequential in terms of number of fields

Reply:
Given that we're now generating metadata files from data dictionaries, I think it's fine not to. Though maybe should have some validation that metadata and df fields actually align?

Reply:
yea filtering via metadata seems like overkill